nixos/doc/manual/release-notes/rl-2305.section.md

```diff
@@ -94,6 +94,8 @@ In addition to numerous new and upgraded packages, this release has the followin
 
   And backup your data.
 
+- `services.dhcpcd` is now started with additional systemd sandbox/hardening options for better security.
```

Comment:
Is this relevant? Do we expect some configurations to break? If not this can be removed.

Reply:
No, a simple notification. Deleted.